### DISCUSSION TITLE
TLS default

### DISCUSSION THREAD

**Discussion by gofaster** [blakeblackshear/frigate#11986] [Ask A Question] [answered]
### What is your question:

In trying to establish a HTTPS connection to the UI, I was unsuccessful until I explictly defined and enabled TLS in the config.
I initially omitted this block because the doc stated it was enabled by default.
```
# Optional: TLS configuration
tls:
  # Optional: Enable TLS for port 8080 (default: shown below)
  enabled: True
```

**Comment by NickM-27** [comment]
It is enabled by default https://github.com/blakeblackshear/frigate/blob/3c12872a56f999cb06cb7a69e8838efc199ba8bc/frigate/config.py#L119

**Comment by gofaster** [comment]
Yes, it's weird.  HTTPS connect attempts with Firefox, Chrome, Edge and curl would fail.  Only HTTP worked.  
HTTPS worked (and HTTP stopped) immediately after I added that config block and restarted the container.

Even when it wasn't explictly defined `/api/config` reported TLS as enabled.  Adding the config block was a desperation move on my part.

**Comment by blakeblackshear** [comment, marked as answer]
I see the problem. I am parsing the file at startup directly, so I dont get the benefit of the default values.
